Makefile

```diff
@@ -1,5 +1,8 @@
 NGINX_PLUS_VERSION=19-1
 NGINX_IMAGE=nginxplus:$(NGINX_PLUS_VERSION)
+NGINX_TEST_ENDPOINT?=http://127.0.0.1:8080/api
```

Comment:
I suggest renaming variables to better reflect their meaning:
- NGINX_TEST_ENDPOINT-> TEST_API_ENDPOINT
- NGINX_TEST_STREAM_ADDRESS -> TEST_UNAVAILABLE_STREAM_ADDRESS
- NGINX_TEST_STREAM_ENDPOINT-> TEST_UNAVAILABLE_STREAM_ENDPOINT 

You also need to add one more as it is missing:
- TEST_API_ENDPOINT_OF_HELPER - the default value should be `http://127.0.0.1:8090` - this is an API endpoint of the second container

Note that all those variables should be exported as environmental variables. That's why you need to export them. Consider doing this:
```
export TEST_API_ENDPOINT_OF_HELPER=http://127.0.0.1:8090
```
Also note there is no need for `?=`.

Reply:
Ok.

I removed 
> NGINX_TEST_STREAM_ENDPOINT

as it is not required.